country-specific/Swedish/SWE_HUNTING_AGRICULTURAL_31.TXT

```diff
@@ -0,0 +1,39 @@
+f=30930000,d=HUNTING & AGRICULTURAL 31 C1
```

Comment:
All the entries in that file have a too long description, they should be <= 30 characters